### DISCUSSION TITLE
How can I enable zstd compression allowed with sled with gluesql

### DISCUSSION THREAD

**Discussion by jorge-alegre** [gluesql/gluesql#253] [Q&A] [answered]
I could use sled with compression. Is built with compression, but seems gluesql is not creating the SledStorage from config where I can use_compression(true).

Is there any work around so I can have db use_compression: true ?

Thanks!

**Comment by panarch** [comment, marked as answer]
To use `sled` config directly, you can use `TryFrom<sled::Config> for SledStorage`.

This example would be helpful.
https://github.com/gluesql/gluesql/blob/main/examples/using_config.rs

By simply changing the config as...
```rust
let config = sled::Config::default()  
    .path("data/using_config")             
    .use_compression(true);       
```
This will work :)

**Reply by jorge-alegre** [reply]
Thanks!

Works!

New to Rust (Coming from other languages and just beginner so far), thing missing was "std::convert::TryFrom".

Thanks a lot for your response.
